src/chi-vue/src/components/data-table/default-config.ts

```diff
@@ -3,7 +3,8 @@ import { DataTableConfig, DataTableModes } from '@/constants/types';
 export const defaultConfig: DataTableConfig = {
   columnResize: false,
   mode: DataTableModes.CLIENT,
-  noResultsMessage: 'No matches found. Please revise search criteria and try again.',
+  noResultsMessage: 'No matching results',
+  noFiltersMessage: 'Search for or select at least one filter to get results',
```

Comment:
Will it make sense to have these messages in a separate constants file?

Reply:
Yea, you're right, thanks!